pkg/daemon/ceph/osd/daemon.go

```diff
@@ -435,6 +435,11 @@ func getAvailableDevices(context *clusterd.Context, agent *OsdAgent) (*DeviceOsd
 		}
 		logger.Infof("device %q is available.", device.Name)
 
+		if device.Type == sys.PartType && agent.storeConfig.EncryptedDevice {
```

Comment:
This is not allowed by ceph-volume? I wonder why the limitation.

Reply:
The recent ceph-volume supports this configuration. However, it may take some time to support this feature in Rook because osd code is very complicated. So I'd like to make a limitation for now. I'll make a feature request and will assign it to myself.